[2024-12-15T22:11:01.752+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: covid_data_pipeline.process_data manual__2024-12-15T22:08:39.174524+00:00 [queued]>
[2024-12-15T22:11:01.769+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: covid_data_pipeline.process_data manual__2024-12-15T22:08:39.174524+00:00 [queued]>
[2024-12-15T22:11:01.773+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-12-15T22:11:01.799+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): process_data> on 2024-12-15 22:08:39.174524+00:00
[2024-12-15T22:11:01.808+0000] {standard_task_runner.py:57} INFO - Started process 332 to run task
[2024-12-15T22:11:01.819+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'covid_data_pipeline', 'process_data', 'manual__2024-12-15T22:08:39.174524+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/covid_pipeline_dag.py', '--cfg-path', '/tmp/tmpwidwo_00']
[2024-12-15T22:11:01.827+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask process_data
[2024-12-15T22:11:02.003+0000] {task_command.py:416} INFO - Running <TaskInstance: covid_data_pipeline.process_data manual__2024-12-15T22:08:39.174524+00:00 [running]> on host efb61c2ab0bb
[2024-12-15T22:11:02.236+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ctw2012105@@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='covid_data_pipeline' AIRFLOW_CTX_TASK_ID='process_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-15T22:08:39.174524+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-15T22:08:39.174524+00:00'
[2024-12-15T22:11:02.335+0000] {covid_processor.py:15} INFO - Cleaning data
[2024-12-15T22:11:02.347+0000] {covid_processor.py:39} INFO - Calculating additional metrics
[2024-12-15T22:11:02.354+0000] {covid_processor.py:61} INFO - Aggregating data
[2024-12-15T22:11:02.403+0000] {python.py:194} INFO - Done. Returned value was: Data processing completed
[2024-12-15T22:11:02.503+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=covid_data_pipeline, task_id=process_data, execution_date=20241215T220839, start_date=20241215T221101, end_date=20241215T221102
[2024-12-15T22:11:02.584+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-15T22:11:02.717+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
